.github/workflows/test.yml

```diff
@@ -24,9 +24,38 @@ env:
   PACKER_VERSION: "1.14.2"
 
 jobs:
+  runners-config:
+    name: Runners Configuration
+    runs-on: ubuntu-latest
+    outputs:
+      runners-config: ${{ steps.runners-config.outputs.runners-config }}
+    steps:
+      - name: Configure runners
+        run: |
+          config="{\"ubuntu-latest\":\"[\\\"runs-on=${{github.run_id}}\\\",\\\"runner=terraform\\\",\\\"tag=${{matrix.stack_slug}}\\\",\\\"extras=s3-cache\\\",\\\"private=false\\\"]\",\"windows-latest\":\"[\\\"windows-latest\\\"]\",\"macos-latest\":\"[\\\"macos-latest\\\"]\"}"
+          echo "runners-config=${config}" >> $GITHUB_OUTPUT
+          echo "runners-config=${config}"
+        id: runners-config
```

Comment:
we already have a matrix for OS. why can't we use that?

Reply:
The matrix configs go to UI. It would looks ugly 

<img width="578" height="324" alt="CleanShot 2025-10-19 at 23 09 45@2x" src="https://github.com/user-attachments/assets/5bfb61d3-ef10-4460-8cff-30ee89e94124" />